spec/lib/tasks/case_contact_types_reminder_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "rails_helper"
+require_relative "../../../lib/tasks/case_contact_types_reminder"
+require "support/webmock_helper"
+
+RSpec.describe CaseContactTypesReminder do
```

Comment:
nit: i think the tests here can be organized better to avoid too much duplication. To me, I feel the contexts are being crammed with too much cases.